lib/livebook/hubs/personal.ex

```diff
@@ -277,4 +277,6 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Personal do
     :ok = Personal.remove_file_system(file_system.id)
     :ok = Broadcasts.file_system_deleted(file_system)
   end
+
+  def deployment_groups(_personal), do: nil
```

Comment:
@josevalim any reason we wouldn't return an empty list?

Reply:
We need to distinguish a hub without deployment groups and one with no deployment groups.

Reply:
We currently only check if it's `!= []`, but yeah.